app/actions/views/channel.js

```diff
@@ -342,6 +342,11 @@ export function toggleDMChannel(otherUserId, visible) {
             category: Preferences.CATEGORY_DIRECT_CHANNEL_SHOW,
             name: otherUserId,
             value: visible,
+        }, {
+            user_id: currentUserId,
+            category: Preferences.CATEGORY_CHANNEL_OPEN_TIME,
+            name: channelId,
+            value: new Date().getTime().toString(),
```

Comment:
Not that it matters that much but you could do `value: Date.now().toString()`

Reply:
Yeah that was odd as well, was trying to be consistent with the redux lib for that preference. I'll make the change.